openapi.yaml

```diff
@@ -255,8 +255,8 @@ components:
 
     UID:
       oneOf:
-      - type: integer
-      - type: string
+        - type: integer
```

Comment:
The indentation here is not YAML-compliant.  This file should be restored to its previous version.